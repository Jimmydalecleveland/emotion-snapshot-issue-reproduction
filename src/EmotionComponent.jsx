import React from 'react'
/** @jsx jsx */
import { css, jsx } from '@emotion/core'
import styled from '@emotion/styled'

const StyledButton = styled.button`
  margin: 0;
  padding: 0;
  background-color: dodgerblue;
  color: white;
`

// 3 different ways to use Emotion CSS-in-JS
const EmotionComponent = () => {
  return (
    <StyledButton id="button2">Some Text</StyledButton>
  )
}

export default EmotionComponent
