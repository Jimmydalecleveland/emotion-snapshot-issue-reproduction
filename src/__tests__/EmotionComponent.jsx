import React from "react";
import { render } from "@testing-library/react";
import EmotionComponent from "../EmotionComponent";

test("demo for snapshot update bug", () => {
  const { container } = render(<EmotionComponent />);
  expect(container).toMatchSnapshot();
});
