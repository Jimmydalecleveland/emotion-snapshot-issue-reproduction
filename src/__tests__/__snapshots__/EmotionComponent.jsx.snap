// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`demo for snapshot update bug 1`] = `
.emotion-0 {
  margin: 0;
  padding: 0;
  background-color: dodgerblue;
  color: white;
}

<div>
  <button
    class="emotion-0"
    id="button1"
  >
    Some Text
  </button>
</div>
`;
